test/src/org/omegat/util/StringUtilTest.java

```diff
@@ -364,4 +366,17 @@ public void testRstrip() {
             // Should fail when stripping null string.
         }
     }
+
+    @Test
+    public void testCaseConversion() {
```

Comment:
Thank you! This is probably the first time someone has supplied a unit test when I asked for one :)